packages/react-native-editor/src/api-fetch-setup.js

```diff
@@ -6,7 +6,7 @@ import apiFetch from '@wordpress/api-fetch';
 
 // Please add only wp.org API paths here!
 const SUPPORTED_ENDPOINTS = [
-	/wp\/v2\/(media|categories|blocks)\/?\d*?.*/i,
+	/wp\/v2\/(media|categories|blocks|themes)\/?\d*?.*/i,
```

Comment:
`getThemeSupports` selector may trigger a request to `/wp/v2/themes?status=active`

Reply:
I'm wondering if we should fetch theme data via the bridge instead of directly making the API request, for other values like colors or gradients, we're getting them from the editor settings:

https://github.com/WordPress/gutenberg/blob/59530f3d5ce3429a50c34e765a55fba84e0202d5/packages/react-native-bridge/ios/Gutenberg.swift#L193-L214

From my POV it's totally ok to fetch from the editor but we might have issues for supporting offline mode, wdyt?

Reply:
I tried fetching it from the bridge on WPiOS via WordPressKit-iOS, but it seems like the [/wp-block-editor/v1/settings]([/wp-block-editor/v1/settings](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/4fb2aac181b808b95e5914efb236ecd7bdb45369/WordPressKit/BlockEditorSettingsServiceRemote.swift#L47)) endpoint is used instead of [/wp/v2/themes](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/4fb2aac181b808b95e5914efb236ecd7bdb45369/WordPressKit/BlockEditorSettingsServiceRemote.swift#L15) if the blog supports it [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/bfd256f79042fd3f8d01ffc09fc86ff248d98c34/WordPress/Classes/Services/BlockEditorSettingsService.swift#L46-L52) and the `/settings` endpoint currently doesn't seem to return the `responsive-embeds` value we need.

Reply:
> I tried fetching it from the bridge on WPiOS via WordPressKit-iOS, but it seems like the [/wp-block-editor/v1/settings](%5B/wp-block-editor/v1/settings%5D(https://github.com/wordpress-mobile/WordPressKit-iOS/blob/4fb2aac181b808b95e5914efb236ecd7bdb45369/WordPressKit/BlockEditorSettingsServiceRemote.swift#L47)) endpoint is used instead of [/wp/v2/themes](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/4fb2aac181b808b95e5914efb236ecd7bdb45369/WordPressKit/BlockEditorSettingsServiceRemote.swift#L15) if the blog supports it [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/bfd256f79042fd3f8d01ffc09fc86ff248d98c34/WordPress/Classes/Services/BlockEditorSettingsService.swift#L46-L52) and the `/settings` endpoint currently doesn't seem to return the `responsive-embeds` value we need.

Right, I checked the schema of `/wp-block-editor/v1/settings` ([reference](https://github.com/WordPress/gutenberg/blob/ac9ab86c129ebc3c20c042413b535abce6ee3c6a/lib/class-wp-rest-block-editor-settings-controller.php#L91-L269)) and there's no reference to `responsive-embeds`, so looks like the theme support settings are only included in the theme model. Now I'm wondering if we should continue fetching the theme via the `api-fetch` or from the theme service ([WordPress-iOS reference](https://github.com/wordpress-mobile/WordPress-iOS/blob/a3eae1d75066b28c215180cafd12e4947c74e556/WordPress/Classes/Services/ThemeService.m#L133-L162) and [WordPressKit-iOS reference](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/4fb2aac181b808b95e5914efb236ecd7bdb45369/WordPressKit/ThemeServiceRemote.m#L21-L46)) and expose it in the RN bridge, wdyt? 

**NOTE:** I noticed that the `Theme` model ([reference](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/develop/WordPressKit/RemoteTheme.h)) doesn't include the theme support settings so, in case we go that way, we should include it.

Reply:
I think the response from [/sites/%@/themes/mine](https://developer.wordpress.com/docs/api/1.1/get/sites/%24site/themes/mine/) also doesn't return the `responsive-embeds` value. I couldn't find any other endpoint that returns it, but could be that I missed out something.
Otherwise I guess we can still fetch it from `/wp/v2/themes` via `BlockEditorSettingsServiceRemote` in WordPressKit-iOS with the function [here](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/4fb2aac181b808b95e5914efb236ecd7bdb45369/WordPressKit/BlockEditorSettingsServiceRemote.swift#L14), even when the blog supports `/wp-block-editor/v1/settings` endpoint. But I think WPiOS is expecting both endpoints to have the same response structure as they are both handed the same completion handler [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/6a28c2f38fdd22b4b09f37b3a255cf622dd9f173/WordPress/Classes/Services/BlockEditorSettingsService.swift#L14). I think this is not the case, but again I may be overlooking something.
This might mean that we may need to create a new model for the `/wp/v2/themes` response along with its local persistence counterparts, which might require some more effort.

Reply:
Good point, I'm afraid that this could turn more complex than what we would expect for this task. At this point, I'd lean towards stick with using the `fetch` as you originally implemented and open an issue as a follow-up to tackle this in the future, wdyt?

Reply:
I agree, opened an issue for this: https://github.com/wordpress-mobile/gutenberg-mobile/issues/3825